editor/src/app/inputs/text-input.service.ts

```diff
@@ -48,12 +48,50 @@ export class TextInputService {
   }
 
   updateField(event) {
+
     if (event instanceof KeyboardEvent && (event.key === 'Escape' || event.keyCode === 27)) {
       (event.target as HTMLInputElement).value = this.lastValue;
       (event.target as HTMLInputElement).blur();
       return null;
     }
 
+    if (event.key === 'ArrowDown' || event.keyCode === 40 || event.key === 'ArrowUp' || event.keyCode === 38) {
+      const bigUnits = ['px', '%', 'pt'];
+      const litleUnits = ['em', 'rem', 'vw', 'vh'];
```

Comment:
`litleUnits -> littleUnits`
or better rename them to `integerUnits`, `decimalUnits`